js/latoken.js

```diff
@@ -0,0 +1,715 @@
+'use strict';
+
+//  ---------------------------------------------------------------------------
+
+const Exchange = require ('./base/Exchange');
+const { ExchangeError, ArgumentsRequired } = require ('./base/errors');
+const { ROUND } = require ('./base/functions/number');
+const crypto = require('crypto')
```

Comment:
We can't use `crypto`, because it's not browser-compatible and it does not work in Python and PHP.

Reply:
done

Reply:
@PolozovT when you finish editing it – just trigger the PR again and it will pass the build phase again and will print out the errors with Travis CI here: 

- https://travis-ci.org/ccxt/ccxt/pull_requests
- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#continuous-integration

Or you can run the checks on your system before submitting to avoid the errors, as described here: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-build--run-tests-on-your-local-machine

Reply:
Make sure the private data structures correspond to https://github.com/ccxt/ccxt/wiki/Requirements